app/controllers/api/v1/public/channels_controller.rb

```diff
@@ -7,4 +7,15 @@ def channels
     end
     render(json: channels_json, status: 200)
   end
-end
\ No newline at end of file
+
+  def totals
+    render(
+      json: {
+        all_channels: Channel.verified.count,
+        twitch_only: Channel.verified.twitch_channels.count,
+        youtube_only:  Channel.verified.youtube_channels.count
```

Comment:
should we add websites? That number should add up to the `totals`

I'm not sure if i'm a big fan of appending `_only` at the end

Reply:
I removed `_only`. `site` wasn't in the requirement document but I'll add anyway.